addons/resource/models/resource.py

```diff
@@ -16,15 +16,13 @@
 from odoo.osv import expression
 from odoo.tools.float_utils import float_round
 
-from odoo.tools import date_utils, float_utils
+from odoo.tools import date_utils
 from .resource_mixin import timezone_datetime
 
 # Default hour per day value. The one should
 # only be used when the one from the calendar
 # is not available.
 HOURS_PER_DAY = 8
-# This will generate 16th of days
-ROUNDING_FACTOR = 16
```

Comment:
We cannot do that in stable since a custom module could import that.

Reply:
Right, I forgot. Nice catch !